hack/update-openapigen.sh

```diff
@@ -23,9 +23,16 @@ if [ -z "${GOPATH:-}" ]; then
 fi
 
 # Update kative files to add `+k8s:openapi-gen=true` to workaround the knative types cannot be load.
-# TBD @jinchihe: This should be updated from knative (PR: https://github.com/knative/pkg/pull/510).
-sed -i '/^type Condition struct {/i // +k8s:openapi-gen=true' vendor/github.com/knative/pkg/apis/condition_types.go
-sed -i '/^type VolatileTime struct {/i // +k8s:openapi-gen=true' vendor/github.com/knative/pkg/apis/volatile_time.go
+# TBD @jinchihe: Remove below after upgraded new knative that includes PR knative/pkg/pull/510.
```

Comment:
since your PR is merged upstream and we also upgraded knative dep to 0.7, can we pin to your commit for knative pkg and remove this hack?

Reply:
Thanks. The PR to upgrade the knative to 0.7 has been merged, but I checked the new knative files today, no the "// +k8s:openapi-gen=true". Seems my PR will be included in next release?

Reply:
I think for knative pkg we can pin to your specific commit for now.

Reply:
Thanks @yuzisun . I just updated the `Gopkg.toml` to my commit for knative/pkg and executed `dep ensure -update github.com/knative/pkg`. Others are generated/updated by dep.